src/cc_message.ml

```diff
@@ -0,0 +1,27 @@
+type cc_message = [ `Cc_restart | `Cc_halt | `Cc_exit ]
```

Comment:
I was curious whether restart and halt should carry a string option for the message?